internal/testutil/testutil.go

```diff
@@ -0,0 +1,13 @@
+package testutil
+
+import (
+	"os"
+	"testing"
+)
+
+// Setenv sets the environment variable, and then unsets it when the test has
```

Comment:
Previous change modified the function name.
```suggestion
// SetEnv sets the environment variable, and then unsets it when the test has
```